app/lib/actions/pulp/repository/refresh.rb

```diff
@@ -7,8 +7,8 @@ class Refresh < Pulp::Abstract
           param :pulp_id
         end
 
-        def plan(repository, input = {})
-          plan_self(:capsule_id => input[:capsule_id], :pulp_id => repository.pulp_id)
+        def plan(repository, options = {})
+          plan_self(:capsule_id => options[:capsule_id], :pulp_id => repository.pulp_id)
```

Comment:
This change doesn't do anything, right?

Reply:
well input is an instance variable in a plan. Don't want params called the same. More a precautionary thing I guess.

Reply:
correct, but input here is overriding the action's input method.

Reply:
so i renamed it for clarity